### PR TITLE
docs: Redisearch Installation 

### PR DESCRIPTION
**Pull request by NagariaHussain**
Updated documentation for Redis 6 and Redisearch module.

### CODE REVIEW

File: frappe_docs/www/docs/user/en/installation.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This guide assumes you are using a personal computer, VPS or a bare-metal server
 ```
   Python 3.6+
   Node.js 12
-  Redis 5                                       (caching and realtime updates)
+  Redis 6                                       (caching and realtime updates)
```
Comment (gavindsouza) [inline frappe_docs/www/docs/user/en/installation.md:32]: System requirements vary depending on Frappe versions. That should be made evident here. Specify that Version 13 requires Redis 6. But then we have develop which has higher Node requirements too....

This should be tracked on it's own separate page. 

Is there a corresponding Frappe PR for this? Or do the older client versions simply work?

**Review by gavindsouza — CHANGES_REQUESTED** [review]

File: frappe_docs/www/docs/user/en/installation.md (hunk 1 above)
Comment (NagariaHussain) [reply frappe_docs/www/docs/user/en/installation.md:32]: Works without any changes on v13.

File: frappe_docs/www/docs/user/en/installation.md [hunk 2]
```diff
@@ -284,3 +295,46 @@ bench start
 ```
 
 Congratulations, you have installed bench on to your system.
+
+## [ERPNext] Optional: Redisearch Redis Module
```
[the remark this answers is not in the document]
Comment (NagariaHussain) [reply frappe_docs/www/docs/user/en/installation.md:299]: I will move this to ERPNext docs. Need to figure out where.

**Comment by NagariaHussain** [comment]
Moved via https://github.com/frappe/erpnext_documentation/pull/375

**Comment by gavindsouza** [comment]
Don't we need to upgrade Redis to v6 too? @NagariaHussain

**Comment by NagariaHussain** [comment]
> Don't we need to upgrade Redis to v6 too? @NagariaHussain 

Yes, it is good we update the version here too. Will create other PR which updates the docs just for Redis 6. On MacOS, its default now.

**Pull request closed without merging**